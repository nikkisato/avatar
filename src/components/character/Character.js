import React from 'react';
import PropTypes from 'prop-types';
import styles from '../characterList/CharacterList.css';

const Character = ({ name, photoUrl }) => {
  return (
    <>
      <section>
        <div className={styles.center}>
          <h3>{name} </h3>
          <img
            src={
              photoUrl
            }
          />
        </div>
      </section>
    </>
  );
};
Character.propTypes = {
  photoUrl: PropTypes.string,
  name: PropTypes.string.isRequired,
  _id: PropTypes.string.isRequired
};

export default Character;
