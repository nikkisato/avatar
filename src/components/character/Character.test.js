import React from 'react';
import { shallow } from 'enzyme';
import Character from './Character';

describe('Character component', () => {
  it('renders Character', () => {
    const wrapper = shallow(<Character />);
    expect(wrapper).toMatchSnapshot();
  });
});
