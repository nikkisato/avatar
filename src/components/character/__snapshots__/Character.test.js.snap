// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders Character 1`] = `
<Fragment>
  <section>
    <div
      className="center"
    >
      <h3>
         
      </h3>
      <img />
    </div>
  </section>
</Fragment>
`;
